__tests__/app.test.js

```diff
@@ -301,4 +301,24 @@ describe("/api/comments/:comment_id", () => {
             expect(response.body.msg).toBe("Comment not found");
         })
     })
+})
+
+describe("/api/users", () => {
+    test("GET: 200 responds with an array of all users in correct formats", () => {
+        return request(app)
+        .get('/api/users')
+        .expect(200)
+        .then((response) => {
+            expect(Array.isArray(response.body.users)).toBe(true);
+            expect(response.body.users).toHaveLength(4);
+            response.body.users.forEach((user) => {
+                 expect(user).toHaveProperty('username');
+                 expect(user).toHaveProperty('name');
+                 expect(user).toHaveProperty('avatar_url');
+                 expect(typeof user.username).toBe('string');
+                 expect(typeof user.name).toBe('string');
+                 expect(typeof user.avatar_url).toBe('string');
```

Comment:
This can be achieved in using the second argument in your first three assertions 😄 
e.g. `expect(user).toHaveProperty('username', expect.any(String));`